docs/manage/ml/_index.md

```diff
@@ -10,31 +10,54 @@ description: "Use Viam's built-in machine learning capabilities to train image c
 ---
 
 Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
-Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
+Common use cases include:
+
+- Object detection and classification which enable smart machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Speech recognition, natural language processing, and speech synthesis, which enable smart machines to verbally communicate with us.
+
+However, your robot can make use of machine learning with nearly any kind of data.
 
 Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
 
-You can [train your own image classification models](/manage/ml/train-model/) or [add an existing model](/manage/ml/upload-model/) for object detection and classification within the platform using data from the [data management service](../../services/data/).
-Object detection and classification models are commonly used to enable robots to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+## Use machine learning with your smart machine
+
+{{< cards >}}
+{{% manualcard %}}
+
+<h4>Train or upload an ML model</h4>
+
+You can [add an existing model](/manage/ml/upload-model/) or [train your own models](/manage/ml/train-model/) for object detection and classification using data from the [data management service](../../services/data/).
```

Comment:
It would have to be this.
```suggestion
You can [train your own models](/manage/ml/train-model/) for object detection and classification using data from the [data management service](../../services/data/) or [add an existing model](/manage/ml/upload-model/).
```

And we did standardize on lower case so that should indeed change.

Reply:
https://github.com/viamrobotics/docs/pull/1984